src/deepsparse/loggers/python_logger.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Implementation of the Python Logger that logs to the stdout
+"""
+import logging
+from typing import Any
+
+from deepsparse.loggers import BaseLogger, MetricCategories
+
+
+logging.basicConfig(format="%(asctime)s %(message)s", datefmt="%m/%d/%Y %I:%M:%S %p")
```

Comment:
I feel that PythonLogger should have a bit of configurability when it comes to the stdout. Maybe having a method to setup the `logging.basicConfig(...)` from `config.yaml`? Something like
```python
def __init__(self, config: Optional[Dict] = None):
    if config:
        self.setup_built_in_logger(config)
        
def setup_build_in_logger(self, config):
    # do something
    args = ...
    logging.basicConfig(args)

Reply:
Disagreed here for now, let's keep things simple and add in features as needed rather than tying to functionality that may or may not be used (and need to make a breaking change on later)

Reply:
Ok, not having strong feelings about this, was just a loose proposal